esp-hal/src/peripheral.rs

```diff
@@ -355,6 +358,7 @@ mod peripheral_macros {
             $(#[$cfg])?
             #[derive(Debug)]
             #[allow(non_camel_case_types)]
+            /// Macro to create a peripheral structure with a virtual interface.
```

Comment:
I don't get this. Isn't the documentation placed on the generated struct, and not the macro?

Reply:
Fixed